components/ws-manager/pkg/manager/create.go

```diff
@@ -379,6 +379,18 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 		}
 		annotations[customTimeoutAnnotation] = req.Spec.Timeout
 	}
+
+	limits := startContext.Class.Container.Limits
+	if limits != nil && limits.CPU != nil {
+		if limits.CPU.MinLimit != "" {
+			annotations[kubernetes.WorkspaceCpuLimitAnnotation] = limits.CPU.MinLimit
```

Comment:
I got confused about the name of annotation(`WorkspaceCpuLimitAnnotation`) and config(`WorkspaceCpuLimitAnnotation`). How about unifying with or without Min? I would recommend adding a Min.